### PR TITLE
Implement flying command

### LINKED ISSUE #687: ENH: Flying as a GM command
### Is your feature request related to a problem?

Most other previous server projects had an option to fly by command (likely because the jetpack objects didn't work). A command is nice to have for ease of use and maintains whatever outfit you had on.

### Describe the solution you'd like

a /fly command, maybe as a toggle. Even better to have Uchu's method of /fly (speed) since the default jetpack speed is slow and clumsy.

### Repository breaking implications

_No response_

### Describe alternatives you've considered

_No response_

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by Jettford**
Resolves #687

### CODE REVIEW

File: dGame/Character.h [hunk 1]
```diff
@@ -425,6 +425,9 @@ class Character {
 	 */
 	const std::vector<LOT>& GetEquippedItems() const { return m_EquippedItems; }
 
+	bool GetIsFlying() { return m_IsFlying; }
+    void SetIsFlying(bool isFlying) { m_IsFlying = isFlying; }
```
Comment (aronwk-aaron) [inline dGame/Character.h:429]: change spaces to tabs, please

**Review by aronwk-aaron — CHANGES_REQUESTED** [review]
Only other thing to note is that docs need to be updated as well

**Review by aronwk-aaron — APPROVED** [review]

**Comment by EmosewaMC** [comment]
What tests have you done for this?

**Comment by Jettford** [comment]
Tests? This adds a command that allows GMs to fly without Jetpacks.

**Comment by EmosewaMC** [comment]
How have you tested this functionality?  What edge cases have been checked
to ensure this doesn't break any current functionality?

**Comment by Jettford** [comment]
Of course I have tested the functionality and tested that it doesn't break anything existing.

**Comment by EmosewaMC** [comment]
> Of course I have tested the functionality and tested that it doesn't break anything existing.

Just wanted to make sure we had some written log of what was tested for the future - looks reasonable to me
